### ISSUE TITLE
A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader:During editing when TAB is pressed and list of list items pops up,Narrator does not announce anything.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [jupyter/notebook#4203]
Environment Details:
Application Name: AzureNotebooks_Jupyter
Login URL: notebooks.azure.com
Browser Details:
Edge: Microsoft Edge 42.17134.1.0
Chrome: Version 70.0.3538.77 (Official Build) (64-bit)
Operating System:Windows10

Steps to reproduce:
1.Login with the appropriate credentials to azure notebook account.    

2.Go to Get started à libraries à Select any of the libraries à click on clone to clone the library

3.Navigate to run button & hit enter --> Navigate to Creation and Deployment of an Azure ML Web Service & hit enter.
4. Navigate to Code Cell block.

Actual
During editing when TAB is pressed and list of list items pops up on the page,Narrator does not announce anything for the end user to understand that various list items are available for the end user to choose from and also when a particualr list item is selected,the narrator does not announce that the list item has been selected.

Expected
During editing when TAB is pressed and list of list items pops up,Narrator does should announce some information about the list items being available for the end users to choose from and also it should announce when a particular list item has been selected.

User Impact
End User will be unaware of the information present on the screen if that is not conveyed by the screen reader.

Recommendation
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

MAS Reference
https://microsoft.sharepoint.com/%3Aw%3A/r/teams/msenable/_layouts/15/WopiFrame2.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D

[A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip](https://github.com/jupyter/notebook/files/2576104/A11y_AzureNotebooks_Jupyter_CreationAndDeployment_ScreenReader.zip)

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
